pkg/client/harbor/harbor.go

```diff
@@ -27,7 +27,7 @@ type Repo struct {
 	username string
 	password string
 
-	helm *helmclassic.Repo
+	Helm *helmclassic.Repo
```

Comment:
You should not expose this in purpose to ease your testing. The `harbor` client is exposing an interface and your tests (the ones under `harbor_test` package) should test the API.

If you need to test the internals, you can define internal tests (under `harbor` package) so you can access this kind of thing.